src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -617,52 +625,189 @@ - (void)_setupSubscription
+        case MTRClusterThreadNetworkDiagnosticsAttributeTxBeaconCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeTxBeaconRequestCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeTxOtherCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeTxRetryCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeTxDirectMaxRetryExpiryCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeTxIndirectMaxRetryExpiryCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeTxErrCcaCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeTxErrAbortCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeTxErrBusyChannelCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxTotalCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxUnicastCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxDataCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxDataPollCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxBeaconCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxBeaconRequestCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxOtherCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxAddressFilteredCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxDestAddrFilteredCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxDuplicatedCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxErrNoFrameCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxErrUnknownNeighborCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxErrInvalidSrcAddrCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxErrSecCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxErrFcsCountID:
+        case MTRClusterThreadNetworkDiagnosticsAttributeRxErrOtherCountID:
+            return YES;
+        default:
+            return NO;
+        }
+    case MTRClusterWiFiNetworkDiagnosticsID:
+        switch (attributePath.attribute.unsignedLongValue) {
+        case MTRClusterWiFiNetworkDiagnosticsAttributeRssiID:
+        case MTRClusterWiFiNetworkDiagnosticsAttributeBeaconLostCountID:
+        case MTRClusterWiFiNetworkDiagnosticsAttributeBeaconRxCountID:
+        case MTRClusterWiFiNetworkDiagnosticsAttributePacketMulticastRxCountID:
+        case MTRClusterWiFiNetworkDiagnosticsAttributePacketMulticastTxCountID:
+        case MTRClusterWiFiNetworkDiagnosticsAttributePacketUnicastRxCountID:
+        case MTRClusterWiFiNetworkDiagnosticsAttributePacketUnicastTxCountID:
+        case MTRClusterWiFiNetworkDiagnosticsAttributeCurrentMaxRateID:
+        case MTRClusterWiFiNetworkDiagnosticsAttributeOverrunCountID:
+            return YES;
+        default:
+            return NO;
+        }
+    case MTRClusterOperationalCredentialsID:
```

Comment:
We should file a followup issue to add ICDManagement ICDCounter, once it gets added to the Darwin APIs.

And CountdownTime in OperationalState.

Reply:
https://github.com/project-chip/connectedhomeip/issues/26805